Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -1044,6 +1044,18 @@
         <item>"Bleiben Sie bis zu Ihrer Genesung trotzdem zu Hause. Falls Sie sich durch eine andere Infektion geschwächt mit dem Coronavirus (SARS-CoV-2) infizieren, kann dies zu schwereren Krankheitsverläufen führen."</item>
         <item>"Gehen Sie nicht krank zur Arbeit, um andere Personen nicht zu gefährden. Falls sich Ihre Symptome verschlechtern, kann die Notwendigkeit eines weiteren SARS-CoV-2-Tests bestehen."</item>
     </string-array>
+    <string name="target_text_subtitle">"Haben Sie sich in den letzten 14 Tagen außerhalb von Deutschland aufgehalten?"</string>
```

Comment:
maybe also prefix with submission, e.g. `submission_target_subtitle`. Also add translation comments like the are used everywhere else.

Reply:
Also applys for the following strings.